notebooks/COS/CalCOS/CalCOS.ipynb

```diff
@@ -8,39 +8,39 @@
     "\n",
```

Comment:
<p>Sierra,</p><p>Thanks for initiating the review process.</p><p>I see this Section 2.3 doesn't exist and linking to a different section.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='60'/>